client/src/components/Sharing.vue

```diff
@@ -82,15 +82,15 @@ import Vue from "vue";
 import BootstrapVue from "bootstrap-vue";
 import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
 import { library } from "@fortawesome/fontawesome-svg-core";
-import { faClipboard, faEdit } from "@fortawesome/free-regular-svg-icons";
+import { faLink, faEdit } from "@fortawesome/free-solid-svg-icons";
```

Comment:
is faEdit available in both regular and solid?

Reply:
Yep. https://fontawesome.com/icons/edit?style=solid